data-prepper-plugins/http-source/src/test/java/org/opensearch/dataprepper/plugins/source/loghttp/HTTPSourceTest.java

```diff
@@ -603,6 +617,82 @@ public void testServerStartCertFileSuccess() throws IOException {
         }
     }
 
+    @Test
+    public void testServerStartCertFileMissing() {
+        try (MockedStatic<Server> armeriaServerMock = Mockito.mockStatic(Server.class)) {
```

Comment:
Similar comment: reduce the amount of code in this `try` block.

Reply:
🫡